tests/e2e-desktop/scripts/test-recycle-bin.ps1

```
@@ -0,0 +1,158 @@
+$HadContent = $PreDeleteContent.Length -gt 0
+
+if ($FileGone -and $HadContent) {
+    Test-Pass "Soft-delete behavior (file removed from mount, had content before deletion)"
+} elseif (-not $HadContent) {
+    Test-Fail "Soft-delete behavior (file had no content before deletion)"
+} else {
+    Test-Fail "Soft-delete behavior (file still on mount after deletion)"
+}
+
+# ---- Cleanup ----
+Write-Host "--- Cleanup ---"
```

PSScriptAnalyzer (1.24.0)

[warning] File 'test-recycle-bin.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)